src/Parser/Reviews/RecentReviewsParser.php

```diff
@@ -62,6 +62,8 @@ function (Crawler $crawler) {
                     if ($crawler->filterXPath('//div[1]/div[1]/div[2]/small')->text() === '(Manga)') {
                         return RecentMangaReview::fromParser(new MangaReviewParser($crawler));
                     }
+
+                    return null;
```

Comment:
@pushrbx Might want to remove this return. IIRC anything returned in the Crawler's `each` call here adds to the array. So this would probably be adding `null` into the array instead.

Reply:
added an `array_filter` call here, so we would keep the null value and adhere to the return value requirement.